clang/test/Sema/enum.c

```diff
@@ -18,6 +19,40 @@ enum x                      // expected-warning {{enumeration values exceed rang
 { y = -9223372036854775807LL-1,  // expected-warning {{ISO C restricts enumerator values to range of 'int'}}
 z = 9223372036854775808ULL };    // expected-warning {{ISO C restricts enumerator values to range of 'int'}}
 
+
+enum GH24667 {   GH24667_x = 9223372036854775807UL, };
+// expected-warning@-1 {{ISO C restricts enumerator values to range of 'int' (9223372036854775807 is too large)}}
+
+#else
+enum e {A,
+        B = 42LL << 32,
+      C = -4, D = 12456 };
+
+enum g {  // too negative
```

Comment:
What does this comment mean? There is no diagnostic here? This also applies to the `too pos` comment below.

Reply:
I have removed those comments, those were copied showing the different behavior in different modes, but those comments should only be in the pre C23 version